python/chatting_sample.py

```diff
@@ -0,0 +1,390 @@
+import json
+import math
+import sys
+import time
+import requests
+import wave
+import threading
+
+try:
+    import azure.cognitiveservices.speech as speechsdk
+except ImportError:
+    print("""
+    Importing the Speech SDK for Python failed.
+    Please run "pip install azure-cognitiveservices-speech" to install the SDK.
+    """)
+    sys.exit(1)
+
+
+# Set up the subscription info for the Speech Service:
+# Replace with your own subscription key and service region (e.g., "westus").
+speech_key, service_region = "YourSubscriptionKey", "YourServiceRegion"
+your_resource_name = "YourResourceName"
+your_deployment_id = "YourDeploymentId"
+api_version = "YourApiVersion"
+your_api_key = "YourApiKey"
+
+
+def read_wave_header(file_path):
+    with wave.open(file_path, 'rb') as audio_file:
+        framerate = audio_file.getframerate()
+        bits_per_sample = audio_file.getsampwidth() * 8
+        num_channels = audio_file.getnchannels()
+        return framerate, bits_per_sample, num_channels
+
+
+def push_stream_writer(stream, filenames):
+    # The number of bytes to push per buffer
+    n_bytes = 3200
+    try:
+        for filename in filenames:
+            wav_fh = wave.open(filename)
+            # Start pushing data until all data has been read from the file
+            try:
+                while True:
+                    frames = wav_fh.readframes(n_bytes // 2)
+                    # print('read {} bytes'.format(len(frames)))
+                    if not frames:
+                        break
+                    stream.write(frames)
+                    time.sleep(.1)
+            finally:
+                wav_fh.close()
+    finally:
+        stream.close()
+
+
+def chatting_from_file():
+    """Performs pronunciation assessment asynchronously from an audio file and enable prosody/content assessment
+        with continuous mode.
+        See more information at https://aka.ms/csspeech/pa"""
+
+    topic = "Talk about your day today"
+    input_files = ["../resources/chat_input_1.wav", "../resources/chat_input_2.wav"]
```

Comment:
Can you change the input wavefiles to our demo waveform?

Reply:
> Can you change the input wavefiles to our demo waveform?

changed